solutions/gold/cf-1666F.mdx

```diff
@@ -0,0 +1,142 @@
+---
+id: cf-1666F
+source: CF
+title: Fancy Stack
+author: Aakash Gokhale
+---
+
+[Official Editorial](https://codeforces.com/contest/1666/attachments/download/15881/nerc-2021-tutorial.pdf)
+
+## Explanation
+
+First we can solve an easier version of the problem where all the blocks are distinct.
+
+Process the blocks from largest to smallest.
```

Comment:
what does this have to do with anything

Reply:
what do you mean?

Reply:
why do we process from big to small

Reply:
Fixed i think? tried to make it more clear